src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -543,20 +556,27 @@ private async Task RefreshKeyValueCollections(CancellationToken cancellationToke
                     });
                 }
 
-                IEnumerable<KeyValueChange> keyValueChanges = await _client.GetKeyValueChangeCollection(
-                    currentKeyValues, 
-                    new GetKeyValueChangeCollectionOptions
-                    {
-                        KeyFilter = changeWatcher.Key,
-                        Label = changeWatcher.Label.NormalizeNull(),
-                        RequestTracingEnabled = _requestTracingEnabled,
-                        RequestTracingOptions = _requestTracingOptions
-                    }, 
-                    cancellationToken).ConfigureAwait(false);
+                try

```

Comment:
Similarly, can we move the `try` to line 538 to just include everything?

Reply:
In this case, we only have one failure point that is covered in the try block. So does it makes sense to include everything?

Reply:
Updated